### PR TITLE
Cat 1073

### PR DESCRIPTION
**Pull request by Olusola-Opencast**

### CODE REVIEW

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/pages/openConditions/PreviousSentencesPage.groovy [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.digital.hmpps.cattool.pages.openConditions
+
+import geb.Page
+
+class PreviousSentencesPage extends Page  {
```
Comment (kberec) [inline integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/pages/openConditions/PreviousSentencesPage.groovy:5]: Just thinking about how we will do this going forward... Would it be the QA who creates these groovy pages? 
Can the existing tests still pass without adding this new page?

File: server/views/formPages/openConditions/previousSentences.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+{% set pageTitle = "Previous sentences" %}
+
+{% block navigation %}
+
+{% include "../../partials/breadCrumbTaskList.html" %}
+
+{% endblock %}
+
+{% set sevenOrMoreYearsTextHtml %}
```
Comment (kberec) [inline server/views/formPages/openConditions/previousSentences.html:15]: Suggest changing to releasedLastFiveYearsTextHtml

File: test/routes/openConditions.test.js [hunk 3]
```diff
@@ -276,3 +281,33 @@ describe('open conditions', () => {
       .expect('Location', `/tasklistRecat/12345`)
   })
 })
+
+describe('Validating Previous Sentences page', () => {
```
Comment (kberec) [inline test/routes/openConditions.test.js:285]: Would this be more suitable in fieldValidation.test.js?

**Review by kberec — CHANGES_REQUESTED** [review]

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/pages/openConditions/PreviousSentencesPage.groovy (hunk 1 above)
Comment (Olusola-Opencast) [reply integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/pages/openConditions/PreviousSentencesPage.groovy:5]: The tests failed without the page which is why I added it. I would prefer not to have to touch these tests at all.

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/OpenConditionsSpecification.groovy [hunk 4]
```diff
@@ -93,6 +95,14 @@ class OpenConditionsSpecification extends AbstractSpecification {
     when: 'the Earliest Release page is completed'
     justifyText << 'details text'
     submitButton.click()
+
+///////////////////////////////////////////////////////////////////////////////
```
[the remark this answers is not in the document]
Comment (Olusola-Opencast) [reply integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/specs/OpenConditionsSpecification.groovy:99]: done

File: integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/pages/openConditions/PreviousSentencesPage.groovy (hunk 1 above)
Comment (Olusola-Opencast) [reply integration-tests/src/test/groovy/uk/gov/justice/digital/hmpps/cattool/pages/openConditions/PreviousSentencesPage.groovy:5]: We need to discuss with Raza the way forward.

File: server/views/formPages/openConditions/previousSentences.html (hunk 2 above)
Comment (Olusola-Opencast) [reply server/views/formPages/openConditions/previousSentences.html:15]: done

File: test/routes/openConditions.test.js (hunk 3 above)
Comment (Olusola-Opencast) [reply test/routes/openConditions.test.js]: Put the validation tests in /config/openConditions.test.js as the validation is in /config/openConditions.js

**Review by kberec — APPROVED** [review]

**Pull request merged**